administration/src/mui-modules/download/DownloadPage.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Typography } from '@mui/material'
+import { styled } from '@mui/system'
+import React, { ReactElement, useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router'
+
+import AppStoreLinks from '../../bp-modules/components/AppStoreLinks'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import { getBuildConfig } from '../../util/getBuildConfig'
+
+const AppIcon = styled('img')({
```

Comment:
🙃 If you don't want to set custom borders, it's also possible to use MUI `Box` with `component='img`  that would there you could use the predefined borders from mui and adjust the colors
But that's really optional
https://mui.com/system/borders/

Feel free to keep your solution